Bottom: a765a457e462570d970410b90af28a0175f38997
Top:    3ecf297a3a83b9a50b744ab2665eb8f700a81f9f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-11 12:13:17 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
